docs/content/en/docs/contribution-guide/_index.md

```diff
@@ -9,13 +9,36 @@ Welcome to Tetragon :) !
 
 We're happy you're interested in contributing to the Tetragon project.
 
+## All contributions are welcome
+
+While this document focuses on the technical details of how to submit patches
+to the Tetragon project, we want to value all kinds of contributions.
+
+For example, actions that can greatly improve Tetragon and contribute to its
+success could be:
+- Write a blog post about Tetragon or one of its use cases, we will be happy to
+  add a reference to it in [resources]({{< ref "/docs/resources" >}}).
+- Talk about Tetragon during conferences or meetups, similarly, as a blog post,
+  video recordings can be added to [resources]({{< ref "/docs/resources" >}}).
+- Share your usage of Tetragon on social platforms, and add yourself to the
+  [user list of the Cilium project](https://github.com/cilium/cilium/blob/main/USERS.md)
```

Comment:
At some point it will probably make sense to have a separate list in this repo, but a shared list is good now I think.

Reply:
yeah I proposed that but better make it grow before splitting.